### PR TITLE
showkey: add page

### PR DESCRIPTION
**Pull request by emiara**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 2.6.4**

### CODE REVIEW

File: pages/linux/showkey.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo showkey -k`
+
+- Display keycodes in ascii, decimal, and hexadecimal:
+
+`sudo showkey -a`
+
+- Exit the program:
+
+`Press Ctrl + d`
```
Comment (vitorhcl) [inline pages/linux/showkey.md:24]:
```suggestion
`Ctrl + d`
```
We indicate keycaps in this way across the pages. Some examples:
- [mpv](https://tldr.inbrowser.app/pages/common/mpv)
- [cmus](https://tldr.inbrowser.app/pages/linux/cmus)

File: pages/linux/showkey.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display scancodes in hexadecimal:
+
+`sudo showkey -s`
+
+- Display keycodes in decimal (default):
+
+`sudo showkey -k`
+
+- Display keycodes in ascii, decimal, and hexadecimal:
```
Comment (vitorhcl) [inline pages/linux/showkey.md:18]:
```suggestion
- Display keycodes in [a]SCII, decimal, and hexadecimal:
```

File: pages/linux/showkey.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# showkey
+
+> Displays the keycode of pressed keys on the keyboard, helpful for debugging keyboard-related issues and key remapping.
+> More information: <https://man7.org/linux/man-pages/man1/showkey.1.html>.
+
+- View keycodes(decimal):
+
+`sudo showkey`
+
+- Display scancodes in hexadecimal:
```
Comment (vitorhcl) [inline pages/linux/showkey.md:10]:
```suggestion
- Display [s]cancodes in hexadecimal:
```

File: pages/linux/showkey.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo showkey -s`
+
+- Display keycodes in decimal (default):
```
Comment (vitorhcl) [inline pages/linux/showkey.md:14]:
```suggestion
- Display [k]eycodes in decimal (default):
```

File: pages/linux/showkey.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# showkey
+
+> Displays the keycode of pressed keys on the keyboard, helpful for debugging keyboard-related issues and key remapping.
+> More information: <https://man7.org/linux/man-pages/man1/showkey.1.html>.
+
+- View keycodes(decimal):
```
Comment (vitorhcl) [inline pages/linux/showkey.md:6]:
```suggestion
- View keycodes in decimal:
```

File: pages/linux/showkey.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# showkey
+
+> Displays the keycode of pressed keys on the keyboard, helpful for debugging keyboard-related issues and key remapping.
```
Comment (vitorhcl) [inline pages/linux/showkey.md:3]:
```suggestion
> Display the keycode of pressed keys on the keyboard, helpful for debugging keyboard-related issues and key remapping.
```

File: pages/linux/showkey.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://man7.org/linux/man-pages/man1/showkey.1.html>.
+
+- View keycodes(decimal):
+
+`sudo showkey`
+
+- Display scancodes in hexadecimal:
+
+`sudo showkey -s`
```
Comment (vitorhcl) [inline pages/linux/showkey.md:12]:
```suggestion
`sudo showkey {{-s|--scancodes}}`
```

File: pages/linux/showkey.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display keycodes in decimal (default):
+
+`sudo showkey -k`
```
Comment (vitorhcl) [inline pages/linux/showkey.md:16]:
```suggestion
`sudo showkey {{-k|--keycodes}}`
```

File: pages/linux/showkey.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display keycodes in ascii, decimal, and hexadecimal:
+
+`sudo showkey -a`
```
Comment (vitorhcl) [inline pages/linux/showkey.md:20]:
```suggestion
`sudo showkey {{-a|--ascii}}`
```

**Review by vitorhcl — APPROVED** [review]
Welcome to tldr! For your first contribution, your page is quite good :)

It is totally normal to miss some things in the first contribution, but your page LGTM after some minor fixes I suggested are resolved.

Thank you for your contribution and for your time!

## Notes
- We always use the imperative mood for the page description when possible. See [style-guide#imperative-mood](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood):
> ## Descriptions
>
> ### Imperative Mood
>
> - **All descriptions must be phrased in the imperative mood.**

- We prefer long options in pages (e.g. --scancodes over -s); if there is an available short option then you can use mnemonics in the description to highlight it inside [] and also leave a placeholder. See https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax

**Review by gutjuri — APPROVED** [review]
Thanks!

**Pull request merged**